build_containers.sh

```diff
@@ -1,8 +1,11 @@
 #!/bin/bash
 
-
 set -euox pipefail
 
+BUILD_ALL=${BUILD_ALL:=0}
+BUILD_NUMBER=${BUILD_NUMBER:=1}
+TAG=${TAG:-test}
```

Comment:
Similarly, I was wondering whether this should default to `test` or something like `$(git branch)` to avoid conflicts with the default values.

Reply:
Again, normally this would be set by Jenkins and would either be latest, release, or the git branch name. Perhaps latest would be a more reasonable default - there's always going to be a latest docker image in the ECR.

Reply:
I guess I'm wondering whether this means we should have defaults or just fail outright if those aren't set since we're only supporting the Jenkins-managed environment.